__tests__/app.test.js

```diff
@@ -263,10 +263,72 @@ describe("News app",()=>{
                 const result = articles.filter(article => {
                     return article.article_id === 1
                 })
-                console.log(result)
                 expect(result[0].comment_count).toBe("11")
             })
         })
+        test("REFACTOR2: status 200, articles default sorted by date descending",()=>{
+            return request(app).get('/api/articles')
+            .expect(200)
+            .then(({body: {articles}})=>{
+                expect(articles).toBeSortedBy("created_at", { descending: true})
+            })
+        })
+        test("REFACTOR2: status 200, allow user to sort on greenlisted fields",()=>{
```

Comment:
Ideally, you'd check the other columns can be sorted by. 

It would be better to catch a bug here, than on the front when you're wondering why the sort_by button isnt working for them all.